### PR TITLE
Deadband

### PR DESCRIPTION
**Pull request by linglejack06**
- add Deadband file in deadband-help package

### CODE REVIEW

**Comment by linglejack06** [comment]
@jkleiber this should cause no issues, i just copy and pasted the code from 2024-robot-code into a file here since i recognized that it would be useful for all projects

**Review by jkleiber — COMMENTED** [review]
@linglejack06 can you rebase this? It looks like it's running the old CI. Also can we put Deadband in the math library, and the other stuff in geometry

**Comment by linglejack06** [comment]
Yeah I'll update that

**Comment by jkleiber** [comment]
@linglejack06 once you resolve merge conflicts here I can take a look at reviewing

**Comment by linglejack06** [comment]
@jkleiber merge conflicts are fixed

File: geometry/build.gradle
Comment (jkleiber) [inline geometry/build.gradle:1]: I don't think this should be deleted

**Review by jkleiber — COMMENTED** [review]
Let's add back the geometry/build.gradle and then this can be merged

File: geometry/build.gradle [hunk 1]
```diff
@@ -6,67 +6,36 @@
...
-	// Allow ignoring certain parts in formatting.
-	toggleOffOn()
-	// apply a specific flavor of google-java-format
-	googleJavaFormat('1.19.2').aosp().reflowLongStrings()
-	// fix formatting of type annotations
-	formatAnnotations()
-}
-}
-
-compileJava.dependsOn 'spotlessApply'
```
Comment (jkleiber) [inline geometry/build.gradle:44]: Can we keep the formatter pls

**Review by jkleiber — COMMENTED** [review]
We still need to add the formatter back to this build.gradle

**Comment by aidnem** [comment]
@linglejack06 can we fix spotless so that the CI will run?

**Comment by linglejack06** [comment]
@aidnem yeah, I think I messed git up because I've tried adding spotless back five times on my local side and it works, but it's apparently not showing up here 😂

File: geometry/src/main/java/coppercore/FieldFinder.java
Comment (jkleiber) [inline geometry/src/main/java/coppercore/FieldFinder.java:1]: This should be in the other PR only

File: geometry/src/main/java/coppercore/Point.java
Comment (jkleiber) [inline geometry/src/main/java/coppercore/Point.java:1]: The geometry PR is leaking into this one somehow, maybe we can merge that first

File: geometry/src/main/java/coppercore/Triangle.java
Comment (jkleiber) [inline geometry/src/main/java/coppercore/Triangle.java:1]: geometry PR

**Review by jkleiber — COMMENTED** [review]
We should remove the geometry PR files from this PR - `git restore <filename>` is probably your friend here

If we get the other PR merged and rebase this one, it will probably ignore those files since they would not have changed. But if anything changes then these files would reappear, so we should just fix it here

File: geometry/src/main/java/coppercore/Triangle.java
Comment (linglejack06) [reply geometry/src/main/java/coppercore/Triangle.java:1]: i think i have removed all geometry-related files other than build.gradle, not sure how that was all happening.

**Review by jkleiber — APPROVED** [review]

**Pull request merged**